app/views/home/index.html.erb

```diff
@@ -1,3 +1,16 @@
 <p style="font-size: 22px;">
   Первый в мире удобный менеджер флеш-карточек. Именно так.
 </p>
+<% if flash[:notice] %>
+  <%= flash[:notice] %>
+<% end %>
+<p> Оригинальный текст карты: </p>
+<% @random_card.to_a.each do |random_card| %>
+  <%= random_card.original_text %>
+  <%= form_tag({controller: "home", action: "check_translation"}) do %>
```

Comment:
if you will use resource route, you should use resource helpers, a-la card_path(@card) http://guides.rubyonrails.org/routing.html